Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM node:7-alpine AS builder
+FROM node:8-alpine
 
-ARG GATEWAY_BASE_URI="'localhost:8080'"
-ARG GATEWAY_USERNAME="admin"
-ARG GATEWAY_PASSWORD="admin"
+# Prepare app directory
+RUN mkdir -p /usr/src/app
+ADD . /usr/src/app
 
-ENV GATEWAY_BASE_URI="${GATEWAY_BASE_URI}"
-ENV GATEWAY_USERNAME="${GATEWAY_USERNAME}"
-ENV GATEWAY_PASSWORD="${GATEWAY_PASSWORD}"
+# Install dependencies
+WORKDIR /usr/src/app
+RUN npm install
```

Comment:
Are we using `npm` or `yarn` @ivanzusko? I see both package-lock.json and yarn.lock in the repo root.

Reply:
Oh yeah, he's off. @italolelis let's use `npm` since you're using node 8 base image (so will use a nice version of npm).

This means you need to do the following:

- Remove yarn.lock 'cos that shit's confusing
- Add (above this line) `npm set progress=false` to speed it up